docs/setup/glue.md

```diff
@@ -0,0 +1,115 @@
+
+This tutorial will cover how to configure both a glue notebook and a glue ETL job. The tutorial is written assuming you
+have a working knowledge of AWS Glue jobs and S3.
+
+In the tutorial, we use
+Sedona {{ sedona.current_version }} and [Glue 4](https://docs.aws.amazon.com/glue/latest/dg/release-notes.html) which runs on Spark 3.3.0, Java 8, Scala 2.12,
+and Python 3.10. We recommend Sedona-1.3.1-incubating and above for Glue.
+
+## Stage Sedona Jar in S3
+
+In an AWS S3 bucket you will need the Sedona-spark-shaded and geotools-wrapper jars. There are two options to get these.
+Ensure the locations of the jars are accessible to your glue job.
+
+### Option 1: Use the Wherobots-hosted jars
+
+[Wherobots](https://wherobots.com/) provides a public S3 bucket with the necessary jars. You can point to these directly in your glue jobs'
+configurations. For {{ sedona.current_version }}, the Sedona and geotools jars are available at the following locations:
+
+* `s3://wherobots-sedona-jars/{{ sedona.current_version }}/sedona-spark-shaded-3.0_2.12-{{ sedona.current_version }}.jar`
```

Comment:
Please also add

```
Please pay attention to the Spark version in the jar name. For Spark 3.0 - 3.3, the name should be `sedona-spark-shaded-3.0`. For Spark 3.4 and 3.5, this will be `sedona-spark-shaded-3.4` and `sedona-spark-shaded-3.5`

Reply:
Ok. I will word it more specifically to glue, since glue is at most 3.3 it will always be the 3.0 version.